compyle/transpiler.py

```diff
@@ -137,14 +138,18 @@ def __init__(self, backend='cython', incl_cluda=True):
         self.blocks = []
         self.mod = None
         self._use_double = get_config().use_double
+        if os.environ.get('COMPYLE_DEBUG') is not None:
```

Comment:
I think, this will evaluate to `True` even if `COMPYLE_DEBUG=0`.

Reply:
Maybe `os.getenv("COMPYLE_DEBUG", 'False').lower() in ['True', 1]`

Reply:
This is also there but adds a dependency: https://www.attrs.org/en/stable/api.html#attrs.converters.to_bool

Reply:
That is intentional, at this point if the variable exists and is set it will treat it, if someone doesn't want it, they can unset the variable.